app/follow/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from album.models import Album
+
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+
+class Follow(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
```

Comment:
Please explain why you used `null=True`. This might lead to corrupted data, unless there's a defined use case.

Reply:
I will review `null=True` and `blank=true` and the difference between them. Actually I had a issue during development. 
To solve that i have used `null=True` 

The behaviour which was causing the issue was somewhat as follows:
When sending a POST request to create an object, I have to send all fields (required as per serializer definition of that object) in the object model. But we cannot send all the fields like user object or album object for instance in the request body. As we are adding user and album while creating an instance of that object in the views.